src/components/Character/styles.scss

```diff
@@ -1,19 +1,184 @@
+// Mobile portrait
 .wrapper {
-	font-size: 1em;
+	display: flex;
+	width: 100%;
+	min-height: 100%;
+	padding: 10%; 		//GUESS
+	position: absolute;
+	top: 0;
+	left: 0;
+	flex-direction: column;
 
 	&[aria-hidden="true"]{
 		opacity: 0;
+		pointer-events: none
 	}
 }
-	.button {
-		&.next {
-			font-size: 1em;
+
+	.container {
+		position: relative;
+		z-index: 1;
+	}
+
+		.content {
+			margin-top: 40vh;
+			max-width: 35em;
 		}
-		&.previous {
-			font-size: 1em;
+	
+			.name {
+				margin-bottom: 0.1em; // GUESS
+				font-size: 5em;
+				font-family: var(--font_family__heading);
+			}
+
+			.nickname {
+				margin-bottom: 0.5em; // GUESS
+				font-size: 2.6em;
+				font-family: var(--font_family__heading);
+			}
+
+			.actor {
+				margin-bottom: 1em; // GUESS
+				font-size: 1em;
+				font-family: var(--font_family__body);
+				font-weight: normal;
+				letter-spacing: 0.25em;
+				text-transform: uppercase;
+			}
+
+			.body {
+				margin-bottom: 4em; // GUESS
+				line-height: 1.5;
+				font-family: var(--font_family__body);
+
+				.paragraph+.paragraph {
+					margin-top: 1em;
+				}
+			}
+
+		.content::after {
+			content: "";
+			display: block;
+			width: 100%;
+			height: 12em;
+			position: fixed;
+			bottom: 0;
+			left: 0;
+			background-image: linear-gradient(
+			    rgba(237, 20, 61, 0) 0%,
+			    rgba(237, 20, 61, 1) 50%,
+			    rgba(237, 20, 61, 1) 100%	
+			);
 		}
+
+		.controls {
+			display: flex;
+			width: 100%;
+			max-width: 25rem; // GUESS
+			position: sticky;
+			bottom: 10vw; // GUESS!
+			justify-content: space-between;
+			font-size: 1.1em;
+			font-family: var(--font_family__heading);
+		}
+
+			.button {
+				display: flex;
+				align-items: center;
+				text-decoration: none;
+				--button_margin: 1em;
+
+				&::before, 
+				&::after {
+					display: inline-block;
+					width: 1em;
+					height: 1.5em;
+					font-size: 1.36em;
+					background-color: lime;
+				}
+
+				&.prev {
+					&::before {
+						content: "";
+						margin-right: var(--button_margin);
+					}
+				}
+
+				&.next {
+					&::after {
+						content: "";
+						margin-left: var(--button_margin);
+					}
+				}
+			}
+
+// Mobile landscape
+@media (orientation: landscape) {
```

Comment:
Ok I managed to do it, but I used translate which is probably a bit sketch. Don't hurt me pls 🙈

Reply:
Fraid so! Big no-no there. It kinda works, but remember that the trick with CSS isn't in getting it to work, it's to have the CSS say what you mean.  We don't mean "wherever the body text is - move it to the right", we mean "lay out the title and the body side-by-side".  It's not actually too painful.  Here's what we have right now:

```
<div class="character__content">
    <h1>Ivar</h1>
    <h2>The Boneless</h2>
    <h3>Alex Hogh Andersen</h3>
    <div class="character__body">
        <p />
        <p />
    </div>
</div>
```

Now, what we want is for the 3 headings to be side-by-side with the body, right? SO, all we need to do is wrap the `<h*>` elements inside a `<div>` and then make `character__content` `display: flex;` and `flex-direction: row` - then you're done!   If you didn't want to add any new `<div>` elements, then you could also use `display: grid` set-up a 2-col 3-row grid and make the body span all the rows.

It was good that you identified that `translate` here was the wrong shout - but the next step there is to take a step back and re-examine the fundamentals of the problem rather than whack-in a solution.  This is a layout problem, and translate is for offsets to occur after the layout, so there's your red flag.

Give's a holla if you need a hand with this!

Reply:
Cool yeah, i was trying to do it without using a new div. I have an irrational fear of using them! xD